.github/pull_request_template.md

```diff
@@ -1,13 +1,40 @@
-
 <!-- In order to keep off topic discussion to a minimum, it helps if the "work to be done" is already agreed on. -->
 <!-- Ideally, a GitHub Issue with the label `Status: Consensus` will have been concluded, and linked to in this PR. -->
-**GitHub Issue:** # / NA
+<!--
+Thanks for submitting a pull request! Please provide enough information so that others can review your pull request.
+-->
+
+**What kind of change does this PR introduce?**
+
+<!-- E.g. a bugfix, feature, refactoring, etc… -->
+
+**Issue Number:**
+
+- Closes # <!-- Replace ___ with the issue number this PR resolves -->
+- Related to # <!-- Use when the PR doesn't completely resolve an issue -->
+- Others? <!-- Add any additional notes or references here -->
+
+**Snapshots:**
+
+<!--Add snapshots or videos wherever possible.-->
 
-**Summary**:
+**If relevant, did you update the documentation?**
+
+<!--Add link to it-->
+
+**Summary**
+
+<!-- Explain the **motivation** for making this change. What existing problem does the pull request solve? -->
+<!-- Try to link to an open issue for more information. -->
+
+**Does this PR introduce a breaking change?**
```

Comment:
Not really sure how there could be a breaking change on a website.  Broken link, maybe?  It doesn't really have programmatic users (where a breaking change might need to be considered).

I'd say we remove this, but can keep it if others think we should.

Reply:
If in future If we have unit testing then might be this is in use.